rust-client/src/strategy_handler/mango.rs

```diff
@@ -0,0 +1,118 @@
+use crate::strategy_handler::base::StrategyHandler;
+use crate::strategy_handler::mango_adapter::{
+    MangoGroupAdapter, MangoNodeBankAdapter, MangoRootBankAdapter,
+};
+use crate::user::get_or_create_ata;
+use anchor_lang::InstructionData;
+use anchor_lang::ToAccountMetas;
+use anyhow::Result;
+use mercurial_vault::strategy::base::{get_mango_group_id, get_mango_program_id};
+
+use solana_program::pubkey::Pubkey;
+use solana_sdk::instruction::AccountMeta;
+use solana_sdk::instruction::Instruction;
+pub struct MangoHandler {}
+
+impl StrategyHandler for MangoHandler {
+    fn withdraw_directly_from_strategy(
+        &self,
+        program_client: &anchor_client::Program,
+        strategy: Pubkey,
+        token_mint: Pubkey,
+        base: Pubkey,
+        amount: u64,
+    ) -> Result<()> {
+        let (vault, _vault_bump) = Pubkey::find_program_address(
+            &[b"vault".as_ref(), token_mint.as_ref(), base.as_ref()],
```

Comment:
We can use the prefix defined in `seed.rs`

Reply:
fixed